lib/kino/hub.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Kino.Hub do
+  @moduledoc """
+  Functions related to hub integrations and Livebook apps.
+  """
+
+  @type app_info ::
+          %{type: :single}
+          | %{:type => :multi, optional(:creator) => user_info()}
```

Comment:
Also, my preference would be towards `:started_by`, since creator *could* perhaps be confused for the notebook "author".

Reply:
I am happy with `:started_by`.